### ISSUE TITLE
Ability to not be notified about a discussion anymore

### ISSUE THREAD

**Issue by matozoid** [theCrag/website#4087]
... or maybe lock it, maybe automatically after a few months or a certain time of inactivity.

This one is just going on and on --> https://www.thecrag.com/discussion/5313018441/female-first-accent-addition

**Comment by nicHoch** [comment]
![image](https://user-images.githubusercontent.com/2226714/174553486-d854991d-627b-4bff-8b57-4270a0240db3.png)

is the "leave" not working for you?

**Comment by matozoid** [comment]
Looks like I already left - but that huge discussion keeps scrolling by every time someone adds a comment :-D So that would mean I am looking for a "mute" option or so.

**Comment by scd** [comment]
I would like to understand what you mean by scrolling by? I could interpret this as seeing the discussions in the streams or getting notified in your messages as having unread messages. 

If you have left the discussion you should not get notification that you have unread messages in the discussion.

However leaving the discussion has no effect in seeing the discussion in the streams. It sounds like this is what you mean as you would not see it as a huge discussion in the unread message notification.

**Comment by lordyavin** [comment]
Maybe the discussion pops up again because someone who you are following participates in the discussion.

**Comment by lordyavin** [comment]
@matozoid is this issue still valid?

**Comment by matozoid** [comment]
Yes, it is what @lordyavin says. It isn't a pressing problem right now - it seems people finally stopped replying.

I'm a bit confused now, isn't there usually a "leave discussion" button somewhere that makes sure you never see it again? That would have been helpful here.
